src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -233,13 +233,13 @@ public partial struct JsonWriterOptions
         public long BytesConsumed { get { throw null; } }
         public int CurrentDepth { get { throw null; } }
         public System.Text.Json.JsonReaderState CurrentState { get { throw null; } }
-        public bool HasValueSequence { get { throw null; } }
+        public readonly bool HasValueSequence { get { throw null; } }
```

Comment:
Here's a recent issue related to this change: https://github.com/dotnet/arcade/issues/3010

Adding `readonly` is fine and is leveraging a new C# 8 feature.